.eslintrc.js

```diff
@@ -11,5 +11,6 @@ module.exports = {
     ecmaVersion: 'latest'
   },
   rules: {
+    'no-console': 'warn'
```

Comment:
please change it to error. Because there are still console.logs left in code base